scripts/pi-hole/js/index.js

```diff
@@ -394,10 +394,16 @@ function updateTopLists() {
   utils.setTimer(updateTopLists, REFRESH_INTERVAL.top_lists);
 }
 
+var previousCount = 0;
 function updateSummaryData(runOnce = false) {
   $.getJSON("/api/stats/summary", function (data) {
     var intl = new Intl.NumberFormat();
-    $("span#dns_queries").text(intl.format(parseInt(data.queries.total, 10)));
+    const newCount = parseInt(data.queries.total, 10);
+    if (previousCount === 0) {
```

Comment:
The issue is here. During startup, if not all queries are loaded completely, `previousCount` might be >0 but `< 1/2 newCount`. But `previousCount` is never updated again, always triggering the update of the chart

![Screenshot at 2024-06-28 21-49-52](https://github.com/pi-hole/web/assets/26622301/5b14435a-d94d-4344-9fcc-24008f4fc5e2)

___

One solution might be just to remove the conditional `if (previousCount === 0)` and always set `previousCount = newCount;`

Reply:
Or add `previousCount = newCount;` in the other `if` (lines 417 to 424).

Reply:
Your correct. If adding it before the comparison check it will never trigger the update....

Reply:
Yes, it should be set *after* the `if`.